eventmesh-runtime/conf/eventmesh.properties

```diff
@@ -71,4 +71,20 @@ eventMesh.registry.plugin.type=namesrv
 eventMesh.server.registry.enabled=false
 
 #prometheusPort
-eventMesh.metrics.prometheus.port=19090
\ No newline at end of file
+eventMesh.metrics.prometheus.port=19090
+
+#trace exporter
+eventMesh.trace.exporter.type=Zipkin
+
+#set the maximum batch size to use
+eventMesh.trace.max.export.size=512
+#set the queue size. This must be >= the export batch size
+eventMesh.trace.max.queue.size=2048
+#set the max amount of time an export can run before getting(TimeUnit=SECONDS)
+eventMesh.trace.exporter.timeout=30
+#set time between two different exports(TimeUnit=SECONDS)
+eventMesh.trace.export.interval=5
```

Comment:
keep the path the same as eventmesh.trace.exporter, eventMesh changed to eventmesh would be better.

Reply:
fixed